doc/man/8/ceph-bluestore-tool.rst

```diff
@@ -171,6 +173,14 @@ Commands
    Changes WAL files from envelope mode to the legacy plain mode.
    Useful for downgrades, or if you might want to disable this new feature (bluefs_wal_envelope_mode).
 
+:command:`create-bdev-labels` --path *osd path* --dev *device*
+
+   Writes a bdev label to BlueStore devices that originally did not support labeling.
+   Reads metadata (e.g., fsid, ceph version) from --path and writes it to the device at --dev.
```

Comment:
Reads metadata (e.g., FSID, Ceph version) from the \`\`--path\`\` argument and writes it to the device specified by the  \`\`--dev\`\` argument.